neon_core/run_neon.py

```diff
@@ -161,9 +161,10 @@ def start_neon():
     if get_neon_device_type() == "server":
         _start_process("neon_core_server")
     else:
+        if get_neon_device_type() not in ("neonPi", "mycroft_mark_2"):
```

Comment:
consider using `is_gui_running` instead, see https://github.com/OpenVoiceOS/ovos_utils/blob/dev/ovos_utils/gui.py#L22

Reply:
that method will detect ovos shell, mycroft-gui and bigscreen.